### PR TITLE
Test travis

### PR DESCRIPTION
**Pull request by Dina-Abd-Elrahman**
add  travis nightly job for builders

https://github.com/threefoldtech/jumpscaleX/issues/138

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,97 +0,0 @@
-# Byte-compiled / optimized / DLL files
```
Comment (zaibon) [inline .gitignore:1]: Why is this removed?

File: .travis.yml [hunk 2]
```diff
@@ -4,11 +4,22 @@ dist: trusty
 services:
   - docker
 
+branch:
+  - test_travis
```
Comment (zaibon) [inline .travis.yml:8]: Does this mean you want to run the test only for this branch? If yes, why?

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (zaibon) [inline .vscode/settings.json:1]: This file should not be part of this PR. Please remove. You can even add this to the gitignore to make sure nobody will push these files anymore

**Review by zaibon — CHANGES_REQUESTED** [review]
Please remove all .pyc files and put back the gitignore file

**Comment by Dina-Abd-Elrahman** [comment]
Okay
Sorry my bad 
I will fix this part and push again

**Pull request closed without merging**